hw/xbox/dsp/dsp.c

```diff
@@ -198,27 +198,47 @@ uint32_t dsp_disasm_address(DSPState* dsp, FILE *out, uint32_t lowerAdr, uint32_
     return dsp_pc;
 }
 
-uint32_t dsp_read_memory(DSPState* dsp, char space_id, uint32_t address)
+uint32_t dsp_read_memory(DSPState* dsp, char space, uint32_t address)
```

Comment:
Ok with the change but in general I think these proxy functions are kind of wonky.

Reply:
Agreed. I don't like them either. For simplicity I kept them in this PR.

I hope that we at least get rid of the 'X, 'Y', 'P' or the constants in the future; the mapping is really stupid. However, I suggest doing that in a cleanup later. For now, the focus is on having at least a rough skeleton for research, debugging and actual audio support.